lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -2736,6 +2750,47 @@ def _add_image_features(self, node_space: schema.NodeSpace) -> None:
         else:
             ...
 
+        if (
+            node_space.disk
+            and node_space.disk.os_disk_type
+            and (
+                (
+                    isinstance(node_space.disk.os_disk_type, schema.DiskType)
+                    and schema.DiskType.Ephemeral == node_space.disk.os_disk_type
+                )
+                or (
+                    isinstance(node_space.disk.os_disk_type, search_space.SetSpace)
+                    and node_space.disk.os_disk_type.isunique(schema.DiskType.Ephemeral)
+                )
+            )
+        ):
+            assert isinstance(node_space.disk, features.AzureDiskOptionSettings)
+            node_space.disk.cached_disk_size = search_space.IntRange(
+                min=self._get_os_disk_size(azure_runbook)
+            )
+
+    def _get_os_disk_size(self, azure_runbook: AzureNodeSchema) -> int:
+        if not azure_runbook:
```

Comment:
Add comments above, why returns zero.

Reply:
Have added some changes. There is no zero returned now.